language/tools/move-package/src/resolution/digest.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use sha2::{Digest, Sha256};
+use std::path::PathBuf;
+
+use crate::source_package::parsed_manifest::PackageDigest;
+
+pub fn compute_digest(paths: &[PathBuf]) -> Result<PackageDigest> {
+    let mut hasher = Sha256::new();
```

Comment:
It looks like the previous hasher, `checksumdir`, [used a different hashing algorithm](https://github.com/sh-zam/checksumdir/blob/45633a625ac19fa068c8eeab976fa2eea849cfca/src/lib.rs#L66): BLAKE2, instead of SHA-2. I presume this doesn't matter for our use case?

Reply:
Correct. For our purposes all that matters is that different file contents will give different hashes. So BLAKE2 vs sha256 etc doesn't matter for our use case here.